app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/points/GeonamesHandlerTest.java

```diff
@@ -222,7 +226,8 @@ public void testGeonamesObjCreationExceptionShortRange() throws IllegalArgumentE
         thrown.expectMessage("BoundingBoxRangeKm can't be null or negative (also not under 100m)");
         Context context = ApplicationProvider.getApplicationContext();
         Assert.assertNotNull(context);
-        UserPoint userPoint = new UserPoint(46.519251915333676,6.558563221333525,220);
+        UserPoint userPoint = UserPoint.getInstance(context);
+        userPoint.setLocation(46.519251915333676, 6.558563221333525, 220, 0);
```

Comment:
Use constants instead
```suggestion
        userPoint.setLocation(MOCK_LOCATION_LAT_LAUSANNE, MOCK_LOCATION_LON_LAUSANNE, MOCK_LOCATION_ALT_LAUSANNE, 0);
```